cocos/2d/CCNode.cpp

```diff
@@ -57,6 +57,7 @@ NS_CC_BEGIN
 
 // FIXME:: Yes, nodes might have a sort problem once every 30 days if the game runs at 60 FPS and each frame sprites are reordered.
 unsigned int Node::s_globalOrderOfArrival = 0;
+int Node::_totalNodeCount = 0;
```

Comment:
`s` prefix is like `m` while s means static member and m means non-static member.
Since we have removed `m` prefix for member variables, should we still keep `s` for static member variables? 
I suggest we uses two underlines (`__`) for static/global variables.